modules/veteran_verification/app/controllers/veteran_verification/v0/disability_rating_controller.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module VeteranVerification
+  module V0
+    class DisabilityRatingController < ApplicationController
+      before_action { authorize :evss, :access? }
+
+      def index
+        response = service.get_rated_disabilities
+        render json: response.rated_disabilities,
+               each_serializer: VeteranVerification::DisabilityRatingSerializer
+      end
+
+      private
+
+      def service
+        EVSS::DisabilityCompensationForm::Service.new(@current_user)
```

Comment:
Interesting, I had glossed over the fact that this was going to EVSS instead of eMIS. Unfortunate that veteran verification will have a dependency on an additional backend service, but I get that this gets us the disability rating date whereas eMIS doesn't.